.github/workflows/check.yml

```diff
@@ -1,9 +1,9 @@
 name: Check
 on:
   push:
-    branches: [master]
+    branches: [master, vange-rs-web]
```

Comment:
since we are already in `vange-rs` repo, perhaps the branch could just be called `web`?